cypress/integration/tx_modal.spec.js

```diff
@@ -0,0 +1,204 @@
+            // Turn the classList into an array
+            const classList = Array.from($div[0].classList)
+            expect(classList).to.include('MuiInputBase-root').and.to.include('Mui-error')
+          })
+
+        // Insert a correct amount
+        cy.get('input[placeholder="Amount*"]').clear().type('0.000002')
+
+        // Form field does not contain an error class
+        cy.get('input[placeholder="Amount*"]')
+          // Parent div is MuiInputBase-root
+          .parent('div')
+          .should(($div) => {
+            // Turn the classList into an array
+            const classList = Array.from($div[0].classList)
+            // Check if it contains the error class
+            expect(classList).to.include('MuiInputBase-root').and.not.to.include('Mui-error')
+          })
+
+        // Click Send max fills the input with token total amount
+        cy.contains('Send max').click()
+        cy.get('input[placeholder="Amount*"]').should('have.value', '0.000004')
+      })
+
+      it('should advance to the Review step', () => {
+        // Clicks Review
+        cy.contains('Review').click()
+
+        // Modal step 2
+        cy.contains('Step 2 of 2').should('be.visible')
+      })
+    })
+
+    describe('Review modal', () => {
+      before(() => {
+        // Wait max 10s for estimate to finish
+        cy.contains('Submit', { timeout: 10000 })
+      })
+
+      it('should have the same parameters as the previous step', () => {
+        // Sender
+        cy.contains('Sending from').parent().next().contains(TEST_SAFE)
+        // Recipient
+        cy.contains('Recipient').parent().next().contains(RECIPIENT_ADDRESS)
+
+        // Token value
+        cy.contains('0.000004 GNO')
+      })
+
+      it('should contain a correctly estimated gas limit value', () => {
+        const GAS_LIMIT = '79804' // gas limit is deterministic
+
+        // Estimated gas price is loaded
+        cy.contains('Estimated fee price').next().should('not.have.text', '> 0.001 ETH')
+
+        // Click Advanced parameters
+        cy.contains('Estimated fee price').click()
+
+        // Find Gas limit
+        cy.contains('Gas limit').next().contains(GAS_LIMIT).should('be.visible')
+
+        // Close info again
+        cy.contains('Estimated fee price').click()
+      })
+
+      it('should contain the Safe nonce upon clicking Advanced parameters', () => {
+        // Click Advanced parameters
+        cy.contains('Advanced parameters').click()
+        // Find Safe nonce
+        cy.contains('Safe nonce').next().contains(SAFE_NONCE).should('be.visible')
+
+        // Close dialog again
+        cy.contains('Advanced parameters').click()
+      })
+
+      it('should initially have a successful simulation', () => {
+        // Simulate
+        cy.contains('Simulate').click()
+
+        // result exists after max 10 seconds
+        cy.contains('The transaction was successfully simulated', { timeout: 10000 })
+      })
```

Comment:
I suggest we take the simulation out of here and put this test to the smoke folder.
Otherwise we'll spend a lot of precious requests.